README.md

```diff
@@ -118,9 +118,23 @@ You can access snapshot builds from the sonatype snapshot repository by adding t
 ```
 
 ## Build
+
+### Requirements
+
+* [Node.Js](https://nodejs.org/en/download/)
+* Java JDK >= 1.8
+* [Lombok Java](https://projectlombok.org) Library (debian pacakge `liblombok-java`)
+* Maven, with the plugins:
+    * [JavaDoc](https://maven.apache.org/plugins/maven-javadoc-plugin/) (debian package `libmaven-javadoc-plugin-java`)
+
+### Environment Variables
+
+* `PATH` must include Node.js's `npm` binary
```

Comment:
Imho this is self-explanatory as node.js is required

Reply:
True; I didn't need to do this, but requirements are useful as a reminder.